docker-compose example/docker-compose.example.yml

```diff
@@ -125,7 +125,7 @@ services:
       - "/etc/timezone:/etc/timezone:ro"   ## use timezone from host
       - "${HOST_PHOTOVIEW_LOCATION}/database/mariadb:/var/lib/mysql" ## DO NOT REMOVE
     healthcheck:
-      test: mysqladmin ping -s -h localhost -u $$MARIADB_USER --password=$$MARIADB_PASSWORD
+      test: healthcheck.sh --connect --innodb_initialized
```

Comment:
For reference of anyone viewing this PR in the future; https://mariadb.com/kb/en/using-healthcheck-sh/